<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchWindow</class>
 <widget class="QMainWindow" name="BatchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>KeyFinder - Batch Analysis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Audio File</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Key</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runBatchButton">
        <property name="minimumSize">
         <size>
          <width>155</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>155</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Run Batch Analysis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Detail_Keyfinder"/>
    <addaction name="actionNew_Batch_Keyfinder"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Window"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionNew_Detail_Keyfinder">
   <property name="text">
    <string>New Detail Keyfinder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_Batch_Keyfinder">
   <property name="text">
    <string>New Batch Keyfinder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionClose_Window">
   <property name="text">
    <string>Close Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
